src/liboslexec/batched_llvm_instance.cpp

```diff
@@ -1024,7 +1024,6 @@ BatchedBackendLLVM::llvm_assign_initial_value(
             // Because we allow temporaries and local results of comparison operations
             // to use the native bool type of i1, we can just skip initializing these
             // as they should always be assigned a value.
-            // TODO: is this equivalent to the old pointer type check? probably not
```

Comment:
Can remove this comment, looks ok, no pointer types, just Symbol stuff.

Reply:
Already removed in the new PR.